File: views/src/main/java/dlgdev/weighttracker/dagger/UserMainActivityComponent.java
package dlgdev.weighttracker.dagger;

import dagger.Component;
import dlgdev.weighttracker.views.users.UserMainActivity;

@Component(modules = {UserMainActivityModule.class, NavigationControllerModule.class,
RepositoriesModule.class})
public interface UserMainActivityComponent {
	void inject(UserMainActivity activity);
}

File: views/src/main/java/dlgdev/weighttracker/dagger/UserMainActivityComponent.kt
package dlgdev.weighttracker.dagger

import dagger.Component
import dlgdev.weighttracker.views.users.UserMainActivity

@Component(modules = arrayOf(UserMainActivityModule::class, NavigationControllerModule::class, RepositoriesModule::class))
interface UserMainActivityComponent {
    fun inject(activity: UserMainActivity)
}
